device/accton/x86_64-accton_as4630_54npe-r0/Accton-AS4630-54NPE/port_config.ini

```diff
@@ -0,0 +1,55 @@
+# name          lanes                 alias             index   speed           autoneg
+Ethernet0       26                    thousandE1        1         1000           1
```

Comment:
autoneg should be either 'on' or 'off'

Reply:
we will update this later.

Reply:
Hi @prgeor 
chihhan123 helps to update the file port_config.ini. please review again.